### PR TITLE
Fix the Travis build

### PR DESCRIPTION
**Pull request by itensionanders**
Make PHP Code Sniffer and PHP Unit Testing pass, by editing code standards and fixing code.

### CODE REVIEW

**Comment by itensionanders** [comment]
Sadly, the pear / pyrus support has been removed from PHP 7.0/nightly in Travis: https://github.com/travis-ci/travis-ci/issues/5256
So we will need to get [PHP_CodeSniffer](http://www.squizlabs.com/php-codesniffer) from some other place, for those. Still works OK for PHP 5.5/5.6

**Comment by itensionanders** [comment]
@jbuchbinder : could you review the new "coding standard" ?
546afe7b529f9cb07fa6aa9556006535d1bc6cb6

Interestingly, the Generic ruleset actually says the following:
`A collection of generic sniffs. This standard is not designed to be used to check code.`

Maybe some of the other standards should be used instead...
http://www.squizlabs.com/php-codesniffer/analysis-of-coding-conventions

**Comment by itensionanders** [comment]
@vvuksan : you might want to have a peek at this, especially the test fix in 113f9f31cb64edbe183a9efd0efae6795172cb10 ?

File: api/host.php [hunk 1]
```diff
@@ -175,7 +176,9 @@ function form_image_url ( $page, $args ) {
       $graph['graph_url'] = form_image_url ( 'graph.php', $graph_arguments );
       $r['graph'][] = $graph;
     } // end foreach metrics
-    if ($debug) { print "<pre>"; print_r($r); die("</pre>"); }
+    if ($debug) { print "<pre>";
```
Comment (jbuchbinder) [inline api/host.php]: (This would be considerably less ugly if indented properly.)

**Comment by jbuchbinder** [comment]
@itensionanders : It looks pretty good to me. There are a few places where the code fixes are a little ugly, but that's it. Is everyone okay with the Makefile task names?

**Comment by itensionanders** [comment]
@jbuchbinder : I'm sure it can be improved manually, I just fixed the warnings/errors - but since the code "standard" doesn't say anything about whitespace, the indentation and trailing whitespace is pretty bad...

i.e. most of the code changes were done *automatically* by "make fix", in 1a684914189d182b12b0c56e06ecc8ea368fb9ae

As for the make targets, both "sniff" (for Code Sniffer) and "fix" (for Code Beautifier and Fixer) names were kinda made up - just wanted to make them short and memorable. And remove logic from Travis config.

The main goal was just to make all the automated tests (Travis CI) pass again. :-)

**Comment by itensionanders** [comment]
Can go with a more boring/crypting "cs" and "cbf", if preferred ? And clean up the whitespace, I suppose.

**Comment by jbuchbinder** [comment]
It's good for now. We can always adjust whitespace and task names if anyone raises any objections.

**Comment by itensionanders** [comment]
Thank you, and yes I'm also sure that some of the stuff that I "excluded" can also be added again - if someone is willing to do the necessary refactorings and cleanup. But now the Travis builds should work again, it was sad that they removed support for *both* of pear and pyrus just like that - but all good now.

**Pull request merged**